src/wp-includes/class-wp-application-passwords.php

```diff
@@ -0,0 +1,318 @@
+<?php
+/**
+ * WP_Application_Passwords class
+ *
+ * @package WordPress
+ * @since   ?.?.0
+ */
+
+/**
+ * Class for displaying, modifying, and sanitizing application passwords.
+ *
+ * @package WordPress
+ */
+class WP_Application_Passwords {
+
+	/**
+	 * The application passwords user meta key.
+	 *
+	 * @since ?.?.0
+	 *
+	 * @type string
+	 */
+	const USERMETA_KEY_APPLICATION_PASSWORDS = '_application_passwords';
```

Comment:
Shouldn't an application password be lock to a single site on a multisite? Shouldn't this be prefixed this site id?

Reply:
Nope.  If someone wants to use the WordPress mobile apps, for example, it would suck for them to log into each individual site in their multisite install individually.